src/main/cc/proxy_client/javac_remote_actions.cc

```diff
@@ -11,15 +15,74 @@ const int kMaxArgsToCheckForJavac = 8;
 const char* kJavacWrapperCommand = "javac_wrapper ";
 const char* kJavacCommand = "javac ";
 
+const char* kJavacWithOptionsCommand = "javac -";
+
 using std::min;
+using std::string;
+using std::set;
+using std::cout;
+
+bool IsAlphabet(const char c) {
+  return (c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z');
+}
+
+string Trim(const string& cmd) {
```

Comment:
I suspect there's also a function for this in absl.

Reply:
I wasn't able to find one for this (I searched the absl c++ repo for trim).

Reply:
Ok there's the Strip functions, but they exactly remove the given prefix/suffix. I want to basically remove all non-alphabets at the beginning and end of the string, so it might be tricky to use strip :(